api_block.go

```diff
@@ -38,24 +43,42 @@ type Block struct {
 	TxCount int `json:"tx_count"`
 
 	// Total output within the block in Lovelaces
-	Output string `json:"output"`
+	Output *string `json:"output"`
```

Comment:
Was it really problematic to just use values instead of having to introduce ptrs?